math/reduce/files/patch-csl-create_old_bundle.diff

```diff
@@ -0,0 +1,10 @@
+--- csl/cslbase/create_old_bundle.sh.orig	2018-09-19 15:28:19.000000000 -0500
++++ csl/cslbase/create_old_bundle.sh	2018-12-19 01:02:57.000000000 -0600
+@@ -26,6 +26,7 @@
+ cp $srcdir/../reduce.doc/*.txt                        $D
+ cp $srcdir/../reduce.doc/*.tex                        $D
+ cp $srcdir/../reduce.doc/*.html                       $D
++cp $srcdir/../reduce.doc/*.gif                        $D
+ F="$1.app/Contents/MacOS/reduce.fonts"
```

Comment:
I used upstream's version of this. Upstream deleted the *.gif copy, otherwise all this is currently in the new version we're using. So I think we don't need this patchfile at all any longer.